samples/core/Saving/Saving/Async/Sample.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EFSaving.Async
+{
+    public class Sample
+    {
+        public static void Run()
+        {
+            using (var db = new BloggingContext())
+            {
+                db.Database.EnsureDeleted();
+                db.Database.EnsureCreated();
+            }
+
+            AddBlogAsync("http://sample.com").Wait();
```

Comment:
Probably not the best idea to show use of .Wait when it is so easy to use this incorrectly and get a deadlock. cc @AndriySvyryd